frontend/kesaseteli/youth/src/pages/_app.tsx

```diff
@@ -0,0 +1,22 @@
+import 'react-toastify/dist/ReactToastify.css';
+
+import Footer from 'kesaseteli/youth/components/footer/Footer';
+import Header from 'kesaseteli/youth/components/header/Header';
+import { getBackendDomain } from 'kesaseteli-shared/backend-api/backend-api';
+import createQueryClient from 'kesaseteli-shared/query-client/create-query-client';
+import { AppProps } from 'next/app';
+import { appWithTranslation } from 'next-i18next';
+import React from 'react';
+import { QueryClientProvider } from 'react-query';
+import BackendAPIProvider from 'shared/backend-api/BackendAPIProvider';
+import BaseApp from 'shared/components/app/BaseApp';
+
+const App: React.FC<AppProps> = (appProps: AppProps) => (
+  <BackendAPIProvider baseURL={getBackendDomain()}>
+    <QueryClientProvider client={createQueryClient()}>
+      <BaseApp {...appProps} header={<Header />} footer={<Footer />} />
```

Comment:
That's interesting! I have always thought the order would only apply in normal spread and wont matter on jsx element, but you seem to be correct: https://stackoverflow.com/questions/42234453/use-spread-props-with-additional-props
I will fix it.. jsut wonder where else this bug is lurking, because I havent care about this anywhere.